crowbar_framework/app/models/network_service.rb

```diff
@@ -312,9 +317,10 @@ def transition(inst, name, state)
         # If we are the admin node, we may need to add a vlan bmc address.
         # Add the vlan bmc if the bmc network and the admin network are not the same.
         # not great to do it this way, but hey.
-        admin_net = Chef::DataBag.load "crowbar/admin_network" rescue nil
-        bmc_net = Chef::DataBag.load "crowbar/bmc_network" rescue nil
-        if admin_net["network"]["subnet"] != bmc_net["network"]["subnet"]
+        node_networks = node.networks
```

Comment:
@vuntz Somehow the installer fails here for me. AFAICS ```node.networks``` is empty because the node doesn't have the network proposal fully applied yet.